### PR TITLE
Activate strictNullChecks for roxfile

### PR DESCRIPTION
**Pull request by ceisele-r**
Issue: EF-1343

### CODE REVIEW

File: roxFile/createroxfile-service.ts [hunk 1]
```diff
@@ -9,6 +9,10 @@ let efAccessToken: string;
 
 function createFileIDField(fileIDFieldName: string, response: any, instance: PH.Instance.IInstanceDetails): void {
   if (fileIDFieldName) {
+    if (instance.extras.fieldContents === undefined) {
+      instance.extras.fieldContents = {};
```
Comment (BCaspari) [inline roxFile/createroxfile-service.ts:13]: would be safer to throw an error here

File: roxFile/roxtrafileapi.ts [hunk 2]
```diff
@@ -179,18 +183,27 @@ export function errorHandling(errorState: number, instance: PH.Instance.IInstanc
...
+      instance.extras.fieldContents = {};
+    }
     instance.extras.fieldContents[PH.tl("ERROR beim Bearbeiten eines Dokumentenfeldes")] = errorField;
     return instance;
   }
 
   errorField = errorHandlingAPICall(errorState, errorField);
   if (errorField.value) {
+    if (instance.extras.fieldContents === undefined) {
+      instance.extras.fieldContents = {};
```
Comment (BCaspari) [inline roxFile/roxtrafileapi.ts:205]: would be safer to throw an error here

**Review by BCaspari — APPROVED** [review]

File: roxFile/createroxfile-service.ts (hunk 1 above)
Comment (ceisele-r) [reply roxFile/createroxfile-service.ts:13]: Thanks

File: roxFile/roxtrafileapi.ts (hunk 2 above)
Comment (ceisele-r) [reply roxFile/roxtrafileapi.ts:205]: Thanks!

**Pull request merged**